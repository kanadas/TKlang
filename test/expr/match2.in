let is_fst = (\x -> match x with 1@_ -> true; _ -> false;) :: int + int -> bool in is_fst (2 @ 5)

