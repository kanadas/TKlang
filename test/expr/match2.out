expr/match2.in

Parse              Failed...

syntax error at line 1, column 36 due to lexer error
